src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -90,11 +113,11 @@ const NavHeader = (props) => {
   };
 
   return (
-    <Animated.View style={{height: animatedHeight, backgroundColor: colors.lightgrey}}>
+    <Animated.View style={styles.container(animatedHeight)}>
       <StatusBar barStyle="dark-content" />
       <View
         source={{uri: coverPath}}
-        style={[styles.Nav(isHeaderHide, topPosition), heightWithCoverImage()]}
+        style={[styles.navContainer(isHeaderHide, topPosition), heightWithCoverImage()]}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `style` prop of the `Animated.View` and `View` components are updated. Make sure these changes align with the desired styling.
```diff
-    <Animated.View style={{height: animatedHeight, backgroundColor: colors.lightgrey}}>
+    <Animated.View style={styles.container(animatedHeight)}>
-        style={[styles.Nav(isHeaderHide, topPosition), heightWithCoverImage()]}
+        style={[styles.navContainer(isHeaderHide, topPosition), heightWithCoverImage()]}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->